File: AngularLovelyjubbly/ClientApp/src/app/shared/models/superbowlodds.ts
import { ITournament } from './tournament';
import { ITeam } from './team';

export interface ISuperbowlOdds {
    superbowlOddsId: number;
    tournament: ITournament;
    team: ITeam;
    odds: number;
}

export class SuperbowlOdds implements ISuperbowlOdds {

    constructor(public superbowlOddsId: number, public tournament: ITournament,
        public team: ITeam, public odds: number) {
    }
}

File: AngularLovelyjubbly.Data.Sql.Models/SuperbowlOdds.cs
namespace AngularLovelyjubbly.Data.Sql.Models
{
    public partial class SuperbowlOdds
    {
        public SuperbowlOdds()
        {
        }

        public int SuperbowlOddsId { get; set; }
        //foreign key
        public int TournamentId { get; set; }
        //navigation property
        public virtual Tournament Tournament { get; set; }
        //foreign key
        public int TeamId { get; set; }
        //navigation property
        public virtual Team Team { get; set; }
        public byte Odds { get; set; }
    }
}
